pkg/subctl/cmd/gather/gather.go

```diff
@@ -116,54 +116,46 @@ func gatherData(cluster *cmd.Cluster) bool {
 		}
 	}
 
-	gatherDataByCluster(cluster, directory)
+	status := cli.NewReporter()
+	status.Start("Gathering information from cluster %q\n", cluster.Name)
+	gatherDataByCluster(cluster, directory, status)
 
-	fmt.Printf("Files are stored under directory %q\n", directory)
+	status.Success("Files are stored under directory %q\n", directory)
+	status.End()
 
 	return true
 }
 
-func gatherDataByCluster(cluster *cmd.Cluster, directory string) {
+func gatherDataByCluster(cluster *cmd.Cluster, directory string, status reporter.Interface) {
 	var err error
 	clusterName := cluster.Name
 
-	fmt.Printf("Gathering information from cluster %q\n", clusterName)
+	clientProducer, err := client.NewProducerFromRestConfig(cluster.Config)
+	if err != nil {
+		status.Failure("Error creating client producer")
+		status.End()
```

Comment:
We want to log the error here. The previous code used `fmt.Printf` but we  can use a status instance to get the red X decoration. 

```suggestion
	        status.Failure("Error creating client producer: %v", err)
```